admin/acceptance.py

```diff
@@ -1466,7 +1466,7 @@ def cluster_cleanup():
     try:
         yield runner.ensure_keys(reactor)
         cluster = yield runner.start_cluster(reactor)
-        if options['distribution'] in ('centos-7', 'rhel-7.2'):
+        if options['distribution'] in ('centos-7', 'rhel-7.2', 'ubuntu-16.04'):
```

Comment:
Because this branch is for distros where we use `journalctl` to capture the remote logs.
And I think RHEL7 isn't in the `DISTRIBUTIONS` because the RHEL7 support isn't completely finished :frowning: